addons/web/static/src/js/core/owl_popover.js

```diff
@@ -0,0 +1,272 @@
+odoo.define('web.OwlPopover', function() {
+    'use strict';
+
+    const { Component, hooks, misc, QWeb } = owl;
+    const { Portal } = misc;
+    const { useExternalListener, useRef, useState } = hooks;
+
+    /**
+     * Popover (owl version)
```

Comment:
Not sure we should specify owl everywhere we use it. It maybe makes sense now, but in 3 years, we'll see that kind of comments everywhere. Looking at the imports, I already know it is an Owl component.

Now that I'm writing this, I see the name of the file and module. I'd rename them to popover.js and web.Popover, for the exact same reasons. We sometimes keep owl in the name, but only when we have both implementations for the same element (for compatibility reasons), and the plan is to rename everything when we'll drop the backward compatibility.

Reply:
agreed, this will get annoying fast otherwise.